base/config_files/docker_run.sh

```diff
@@ -91,6 +91,9 @@ if [ ! -f ./config/settings.inc.php ] && [ ! -f ./install.lock ]; then
 
         if [ $? -ne 0 ]; then
             echo 'warning: PrestaShop installation failed.'
+        else
+            echo "\n* Removing install folder..."
+            rm -r /var/www/html/$PS_FOLDER_INSTALL/
```

Comment:
@Quetzacoalt91 @PierreRambaud I know you already approved this, but removing the whole folder might be troublesome in some cases (like if you want to debug the installation in a docker for instance 😅 )
So I would suggest an alternative to just rename it, this way it's easy to fallback on the still existing folder
```suggestion
            mv /var/www/html/$PS_FOLDER_INSTALL/ /var/www/html/_$PS_FOLDER_INSTALL/
```

Reply:
Not a good idea to keep the install directory after having a successful installation. Even with a `_`, if someone found the URL, he can reinstall/break your shop :/

This directory is removed only if the installation process is ok, so there is nothing to debug when everything is ok :D